source/_includes/installation/operating_system.md

```diff
@@ -396,15 +396,40 @@ Minimum recommended assignments:
 
 {% if page.installation_type == 'windows' or page.installation_type == 'Linux' %}
 
-- title: Vmware Workstation
+- title: VMware Workstation
   content: |
-    1. Create a new virtual machine.
-    2. Select **Custom**, make it compatible with the default of Workstation and ESX.
-    3. Choose **I will install the operating system later**, select **Linux** > **Other Linux 5.x or later kernel 64-bit**.
-    4. Select **Use Bridged Networking**.
-    5. Select **Use an existing virtual disk** and select the VMDK file above.
+    1. Start VMware Workstation and select **Create a New Virtual Machine**.
+       - Note: the exact name and location of the settings below depend on the VMware version. This procedure is based on version 17.
+    2. Select **I will install the operating system later**, then select **Linux** > **Other Linux 5.x kernel 64-bit**.
+    3. Give the VM a name, `home-assistant`, and define an easy to reach storage location, such as `C:\home-assistant`.
+    4. Specify the disk size and select **Store virtual disk as a single file**.
+    5. Select **Customize Hardware**.
+    6. Define the amount of memory and the number of cores the VM is allowed to use.
+    7. Remove the **New CD/DVD** entry. It will not be used.
+    8. Connect an Ethernet cable and make sure it is connected to your network.
+    9. Under **Network adapter**, select **Bridged: Connected directly to the physical network**.
+       - Make sure **Replicate physical network connection state** is not selected.
+       - Select **Configure Adapters**.
+       - Make sure all virtual adapters and Bluetooth devices are deselected.
+       - Select your host network adapters. Most likely, they are the first 2 checkboxes in the list:
+         - One for Ethernet and one for Wi-Fi.
```

Comment:
I'd suggest to only bridge one of those. Also, we should encourage to use Ethernet whenever possible.